lib/dogma.ex

```diff
@@ -10,13 +10,33 @@ defmodule Dogma do
   alias Dogma.Rules
   alias Dogma.ScriptSources
 
-  def run(dir_path \\ "") do
-    dir_path
+  @formatters %{
+    "simple" => Formatter.Simple,
+    "emacs"  => Formatter.Emacs
+  }
+
+  def run(argv) do
```

Comment:
When I looked at the original code it felt like the mix task was mostly just delegating to the dogma run method. Though now that I think about it the mix task does seem like a better place. Do you want me to break out the option parsing into it's own module as well?